PROFILER.md

```diff
@@ -9,21 +9,24 @@ which inspects the state of all threads to determine how long they have been
 running for. This does not require any code to be modified to support profiling
 so that any code which runs will be profiled automatically.
 
-The profiler can be enabled by setting the following:
+## Enabling Profiling
 
- * `com.iopipe.plugin.profiler` to `true`
- * `IOPIPE_PROFILER_ENABLE` to `true`
+### In General:
```

Comment:
The "in general" is redundant ... you're telling people they can set the environment variable, but then the following header is for serverless framework specifically